### DISCUSSION TITLE
zeek intel alerts

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2449] [Unsupported Versions] [answered]
Is there a way to generate alerts on zeek intel matches ? Wondering if elastalert is the way to go and how to do so.

**Comment by TOoSmOotH** [comment, marked as answer]
I think elastalert would be the way for now.

**Reply by 0nnyx** [reply]
OK, follow up question then. Having a working yaml elastalert rule, how do I make the matches/sent-alerts visible on the Alert tab of SO ?
Right now, the alerts are only logged in the `*:elastalert_status*` kibana index. Is there a specific alerter to use for them to show up in that tab (using the debug alerter right now) ? Thx

**Reply by weslambert** [reply]
To actually have an alert viewable in the Alerts pane, it would need to be of `event.dataset`: `alert`, and you would want to have a `event.severity` associated with it.  We don't currently cast Zeek intel/Notice matches as alerts, but may consider doing it in the future, or maybe even particular alerts generated by Elastalert.  To get around this for now, you could build a custom Elastalert alerter, or cut out Elastalert, and have Zeek intel matches recategorized as an alert via the Inget Node pipeline(s).
